cherro-front/.eslintcache

```diff
@@ -0,0 +1 @@
unt":0,"fixableWarningCount":0,"usedDeprecatedRules":"56"},{"filePath":"93","messages":"94","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"56"},{"filePath":"95","messages":"96","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"56"},{"filePath":"97","messages":"98","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"56"},{"filePath":"99","messages":"100","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"56"},{"filePath":"101","messages":"102","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"56"},{"filePath":"103","messages":"104","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"56"},{"filePath":"105","messages":"106","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/home/nicole/cherroapp/cherro-front/src/index.js",[],["107","108"],"/home/nicole/cherroapp/cherro-front/src/serviceWorker.js",[],"/home/nicole/cherroapp/cherro-front/src/app.jsx",[],"/home/nicole/cherroapp/cherro-front/src/route-components.js",[],"/home/nicole/cherroapp/cherro-front/src/routes/index.js",[],"/home/nicole/cherroapp/cherro-front/src/common/error-boundary/index.js",[],"/home/nicole/cherroapp/cherro-front/src/routes/routes.js",[],"/home/nicole/cherroapp/cherro-front/src/routes/route-helpers.js",[],"/home/nicole/cherroapp/cherro-front/src/pages/home/index.js",[],"/home/nicole/cherroapp/cherro-front/src/pages/not-found/index.js",[],"/home/nicole/cherroapp/cherro-front/src/routes/app-redirect.jsx",[],"/home/nicole/cherroapp/cherro-front/src/routes/router.jsx",[],"/home/nicole/cherroapp/cherro-front/src/routes/app-link.jsx",[],"/home/nicole/cherroapp/cherro-front/src/common/error-boundary/error-boundary.jsx",[],"/home/nicole/cherroapp/cherro-front/src/helpers/params-helper.js",[],"/home/nicole/cherroapp/cherro-front/src/pages/home/home.jsx",[],"/home/nicole/cherroapp/cherro-front/src/pages/not-found/not-found.jsx",[],"/home/nicole/cherroapp/cherro-front/src/helpers/logger.js",[],"/home/nicole/cherroapp/cherro-front/src/hocs/with-layout/index.js",[],"/home/nicole/cherroapp/cherro-front/src/routes/scroll-to-top/index.js",[],"/home/nicole/cherroapp/cherro-front/src/pages/unexpected-error/index.js",[],"/home/nicole/cherroapp/cherro-front/src/config/constants.js",[],"/home/nicole/cherroapp/cherro-front/src/hocs/with-layout/layout.jsx",[],"/home/nicole/cherroapp/cherro-front/src/hocs/with-layout/with-layout.jsx",[],"/home/nicole/cherroapp/cherro-front/src/routes/scroll-to-top/scroll-to-top.jsx",[],"/home/nicole/cherroapp/cherro-front/src/pages/unexpected-error/unexpected-error.jsx",[],{"ruleId":"109","replacedBy":"110"},{"ruleId":"111","replacedBy":"112"},"no-native-reassign",["113"],"no-negated-in-lhs",["114"],"no-global-assign","no-unsafe-negation"]
```

Comment:
I don't think we should track this file.

Reply:
This file is still here. You need to explicitly delete it before adding it to the gitignore file 😄

Reply:
oopsi, okay!